understanding/adaptable.html

```diff
@@ -54,79 +52,63 @@
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				Guideline 1.3:</span>Adaptable
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				ガイドライン 1.3:</span>適応可能
+
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Guideline </header>
+               <header class="box-h  box-h-icon">ガイドライン</header>
                <div class="box-i">
-                  <p>Create content that can be presented in different ways (for example simpler layout)
-                     without losing information or structure.
+                  <p>情報、及び構造を損なうことなく、様々な方法 (例えば、よりシンプルなレイアウト) で提供できるようにコンテンツを制作すること。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The purpose of this guideline is to ensure that all information is available in a
-                  form that can be perceived by all users, for example, spoken aloud, or presented in
-                  a simpler visual layout. If all of the information is available in a form that can
-                  be determined by software, then it can be presented to users in different ways (visually,
-                  audibly, tactilely etc.). If information is embedded in a particular presentation
-                  in such a way that the structure and information cannot be programmatically determined
-                  by the assistive technology, then it cannot be rendered in other formats as needed
-                  by the user. 
+               <h2>意図</h2>
+               <p>このガイドラインの目的は、例えば、音声で読み上げたり、より単純なレイアウトで提示したりというように、すべての情報をすべての利用者が知覚できるように提供することである。すべての情報が、ソフトウェアによって解釈できる形式で利用可能な場合、情報を様々な方法で (視覚的、聴覚的、触覚的など) 利用者に提示できる。構造及び情報が支援技術経由でプログラムによる解釈ができないような、特定の提示で情報が埋め込まれる場合、その情報は、利用者が必要とするその他の形式でレンダリングできない。
                   
                </p>
-               <p>The Success Criteria under this guideline all seek to ensure that different types
-                  of information that are often encoded in presentation are also available so that they
-                  can be presented in other modalities. 
+               <p>このガイドラインに属する達成基準はすべて、他のモダリティで提示できるようにするために、提示で埋め込まれている様々な種類の情報も利用可能にすることを保証するよう努めるものである。
                   
                </p>
                <ul>
                   
                   
-                  <li>
-                     							        the way the parts of a Web page are organized in relation to each other;
-                     and the way
-                     a collection of Web pages is organized
+                  <li>ウェブページの各部分が互いに関してどのように組織されるか、及びウェブページの集合がどのように組織されるか。
```

Comment:
ここもなかなか意味不明な感じですね。
このガイドラインが何を述べているのか、というようなことを書いているのですが、今の訳だと何を言っているのかわかりにくい感じがします。"the way" = 方法、と言い切ってやると良いかも。
あと、原文の末尾にピリオドがないので句点はいらないですね。

```suggestion
                  <li>ウェブページの各部分の関連性を組織化する方法、及びウェブページの集合を組織化する方法
```